src/legacy-routes.js

```diff
@@ -154,6 +231,7 @@ const unspentTxOutputs = (dbApi: any, { logger, apiConfig }: ServerConfig) => as
     const coins = row.cuCoins
     const newRow = row
     newRow.cuCoins = { getCoin: coins }
+    newRow.cuId = row.cuId // TODO/hrafn \x format
```

Comment:
this seems redundant given that row is assigned to newRow two lines above

Reply:
Yes, before, I unwrapped the cuId (hash) due to the fact that hashes are not stored in the db as previously, but have '\x' prefix. When I tried sending our old transactions from healthcheck, the cardano-submit-api complained that it can't find transaction e.g. 'b9ec5...'. But when I queried the database for '\xb9ec5', it found it. For now, I concluded that they hide their utxo functionality and don't bother with unwrapping the hash when signing transactions, therefore we will most likely need to return the \x format to frontend where it signs it this way so that cardano-submit-api is able to find such hashes. The TODO is there to remind me of this.

Reply:
I'm not sure if I understand completely, but isn't there a way to "shield" frontend from this encoding overhead by handling it directly within the logic of the API calls?

Reply:
Submit api destructures the signed transactions we send it and validates it by querying the database from within the service. Therefore we need to encode it in the correct format. For now, the format without '\x' did not work. We'll be wiser when we are able to sign the transactions after we get raw txs working.

Reply:
You can indeed disable the integrity check in the frontend: https://github.com/vacuumlabs/adalite/blob/develop/app/frontend/wallet/byron/cardano-wallet-secret-crypto-provider.ts#L57 - for "mnemonic" mode, so you don't need to depend on the raw tx just to try out submission of new txs